File: Arrays/linearSearch.cpp
#include <iostream>
using namespace std;

int linear_search(int array[], int n, int x)
{
    for (int i = 0; i < n; i++)
    {
        if (x == array[i])
        {
            return i;
        }
    }
    return -1;
}

int main()
{
    int array[] = {1, 55, 66, 43, 26, 94};
    int size = (sizeof array) / (sizeof array[0]);

    cout << linear_search(array, size, 66) << endl;
    cout << linear_search(array, size, 5) << endl;
    return 0;
}

File: Arrays/linear_search.py
def linear_search(arr, target):
    for i in range(0, len(arr)):
        if arr[i] == target:
            return i
    return -1

if __name__ == '__main__':
    arr = [5, 3, 8, 4, 2]
    target = 2
    result = linear_search(arr, target)
    if result != -1:
        print(f"Element found at index: {result}")
    else:
        print("Element not found")